### DISCUSSION TITLE
In GDB, how can I find out which element is currently being calculated and its material information?

### DISCUSSION THREAD

**Discussion by Yasunaoka** [idaholab/moose#27953] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello moose community! I'm encountering a bit of numerical problems with my current models, so I'd like to check step by step how they change.
I've found that ```gdb``` might meet my needs, but what's in the debugging page isn't quite enough for me.
I've now got ```gdb``` running successfully and can perform breakpoints and stepping where I want.
However, when I stop at a particular location in a particular file and want to query some data about the current computational process (e.g. ```_current_elem```,```_displacement```,```_stress```, etc.), I don't really know how to get them.
I did query the relevant commands, such as ```info locals```, ```info args```, ```info variables```, but I didn't find any of the above required data. Can anyone help me with this? Any suggestions are much appreciated!
Sincerely.
Sen Huang

**Comment by GiudGiud** [comment, marked as answer]
Hello

These will be available in the computeValue / computeQpProperty etc, the innermost call in the element loops.
They wont be elsewhere and they wont be initialized.
```
info locals
```
should work. Else you can use `p <name of variable>` iirc

Guillaume
